src/cljs/commiteth/routes.cljs

```diff
@@ -14,12 +14,18 @@
   "A function which will be called on each route change."
   [name params query]
   (println "Route change to: " name params query)
-  (rf/dispatch [:set-active-page name]))
+  (rf/dispatch [:set-active-page name params query]))
 
 (defn setup-nav! []
   (bide/start! router {:default     :bounties
                        :on-navigate on-navigate}))
 
-(defn nav! [route-id]
-  (bide/navigate! router route-id {}))
+(defn nav!
+  ([route-id]
+   (nav! route-id {}))
+  ([route-id params]
+   (nav! route-id params {}))
```

Comment:
Use `nil` instead of empty maps here to avoid stale `?` at the end of the URI.

Reply:
Done!